test/Polar/gradient.jl

```diff
@@ -85,16 +85,18 @@ function test_line_flow_gradient(polar, device, MT)
         ExaPF.flow_constraints(polar, g, adcache)
         return sum(g)
     end
-    # adgradg = ForwardDiff.gradient(sum_constraints,x)
+    adgradg = ForwardDiff.gradient(sum_constraints,x)
```

Comment:
Right, we should uncomment this line. This line was commented previously as evaluating `ForwardDiff.gradient` on `sum_constraints` breads with `CUDA.@allowscalar false`

Reply:
Yes, so I allowed it now. Or should we disable the test with CUDA. In general, it's good to at least test here with AD on the CPU.